pages/FilePage.js

```diff
@@ -45,7 +45,13 @@ export function FilePage() {
         await ImagePicker.requestMediaLibraryPermissionsAsync();
 
       if (permissionResult.granted === false) {
-        Alert.alert("Permission to access camera roll is required!");
+        Toast.show({
+          type: "error",
+          text1: "Error!",
+          text2: `Permission to access the camera roll is required!`,
```

Comment:
@filiptronicek Do you think backtick is needed here? If you see `text1` its in dquote 🤔